src/duckling/controls/layer-dialog.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
```

Comment:
This file doesn't belong in controls/. It should be in the same folder as the layers service.